src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -337,4 +343,76 @@ public static ProjectListOption fields(ProjectField... fields) {
    *     Platform Services</a>
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
+
+  /**
+   * Clears a Policy from a resource.
+   *
+   * @throws ResourceManagerException upon failure
+   * @see <a href=
+   *     "https://cloud.google.com/resource-manager/reference/rest/v1/folders/clearOrgPolicy">Resource
+   *     Manager clearOrgPolicy</a>
+   */
+  void clearOrgPolicy(String resource, ClearOrgPolicyRequest request);
+
+  /**
+   * Gets the effective Policy on a resource
+   *
+   * <p>This is the result of merging Policies in the resource hierarchy. The returned Policy will
+   * not have an etag set because it is a computed Policy across multiple resources. Subtrees of
+   * Resource Manager resource hierarchy with 'under:' prefix will not be expanded.
+   *
+   * @throws ResourceManagerException upon failure
+   * @see <a
+   *     href="https://cloud.google.com/resource-manager/reference/rest/v1/folders/getEffectiveOrgPolicy">Resource
+   *     Manager getEffectiveOrgPolicy</a>
+   */
+  OrgPolicy getEffectiveOrgPolicy(String resource, GetEffectiveOrgPolicyRequest request);
+
+  /**
+   * Gets the Policy on a resource.
+   *
+   * <p>If no Policy is set on the resource, a Policy is returned with default values including
+   * POLICY_TYPE_NOT_SET for the policy_type one of. The etag value can be used with
+   * projects.setOrgPolicy() to create or update a Policy during read-modify-write.
+   *
+   * @throws ResourceManagerException upon failure
+   * @see <a
+   *     href="https://cloud.google.com/resource-manager/reference/rest/v1/folders/getOrgPolicy">Resource
+   *     Manager getOrgPolicy</a>
+   */
+  OrgPolicy getOrgPolicy(String resource, GetOrgPolicyRequest request);
+
+  /**
+   * Lists all the Constraints that could be applied on the specified resource.
+   *
+   * @throws ResourceManagerException upon failure
+   * @see <a
+   *     href="https://cloud.google.com/resource-manager/reference/rest/v1/folders/listAvailableOrgPolicyConstraints">Resource
+   *     Manager listAvailableOrgPolicyConstraints</a>
+   */
+  Page<Constraint> listAvailableOrgPolicyConstraints(
+      String resource, ListAvailableOrgPolicyConstraintsRequest request);
+
+  /**
+   * Lists all the Policies set for a particular resource.
```

Comment:
all the polices? The why the paging?

Reply:
because it will return the instance of ListOrgPoliciesResponse if successful, you can see here what ListOrgPoliciesResponse contains
https://cloud.google.com/resource-manager/reference/rest/v1/ListOrgPoliciesResponse

Reply:
I think you need to delete "all" here. It's confusing